packages/valory/skills/apy_estimation_abci/models.py

```diff
@@ -81,7 +189,12 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
         self.optimizer_params = self._ensure("optimizer", kwargs)
         self.testing = self._ensure("testing", kwargs)
         self.estimation = self._ensure("estimation", kwargs)
-        self.pair_ids = self._ensure("pair_ids", kwargs)
+        pair_ids = self._ensure("pair_ids", kwargs)
+        self.pair_ids: PairIdsBackwardsCompatibleType = (
+            _pair_ids_backwards_compatible(pair_ids)
+            if kwargs.pop("backwards_compatible", True)
+            else pair_ids
+        )
```

Comment:
This makes the changes compatible with our `FetchBehaviour` implementation, by keeping only the SpookySwap's pair ids. After merging this, I will open another PR, which will remove this and update the behaviour to work with the new configuration, and therefore any subgraph that follows the same format.